src/main/java/frc/robot/subsystems/IndexerSubSystem.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMax.ControlType;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import frc.robot.config.Config;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class IndexerSubSystem extends SubsystemBase {
+  
+  //Instance Variables
+  private CANSparkMax m_indexer;
+  private SparkMaxPIDController m_pidController;
+  private RelativeEncoder m_encoder;
+  private double targetRPM;
```

Comment:
Set to an initial value of targetRPM here, e.g. 200.
Later, we can move both targetRPM and incrementalPosition to config.java.